### PR TITLE
+ Adobe's React Spectrum

### PR DESCRIPTION
**Pull request by tomByrer**
https://github.com/adobe/react-spectrum/blob/c700898916bbd076bcc63e49d77c16d80623a8e7/packages/%40react-spectrum/radio/docs/RadioGroup.mdx

Glad to see your hard work making an impact.

### CODE REVIEW

File: sites.json [hunk 1]
```diff
@@ -155,6 +155,15 @@
     "categories": [
       "docs",
       "blog"
+    ],
+  {
```
Comment (ChristianMurphy) [inline sites.json]: There appears to be a missing closing brace here.

**Review by ChristianMurphy — COMMENTED** [review]
Thanks @tomByrer!
A quick change to make CI happy and this will LGTM.

File: sites.json (hunk 1 above)
Comment (tomByrer) [reply sites.json]: TY I see where I got confused.

**Review by ChristianMurphy — APPROVED** [review]

**Comment by johno** [comment]
Thanks!

**Pull request merged**